### ISSUE TITLE
Postgres "timestamp with time zone" columns do not show time and not able to query according to time

### ISSUE THREAD

**Issue by maxbeatty** [metabase/metabase#2277]
I have some ["timestamp with time zone" in postgres](http://www.postgresql.org/docs/9.1/static/datatype-datetime.html) because that's [how sequelize defines them](http://sequelize.readthedocs.org/en/latest/docs/models-definition/#data-types). If I run this query in `psql`, I get `2016-03-30 18:26:18.949+00` as the result.

```
select created_at from races order by created_at desc limit 1;
```

If I look at that same record in Metabase, I'm shown `2016-03-30T00:00:00.000Z` with all the time information zero'd out instead of with `18:26:18.949` from postgres. If I try to sort a query by a column like this, it does not appear to use the underlying time information either because the order does not change. Queries for these record by "day" work as expected.

*  I am using the Chrome 49 browser.
*  My computer's OS is OS X 10.11.4.
*  I'm running Metabase on AWS.
*  My database is postgres 9.4.4.
*  My Metabase version is 0.16.0.

May be related to #1576 and #1929. Please let me know what other information I can provide.

**Comment by agilliland** [comment]
okay, i'll try reproducing this today.

**Comment by tlrobinson** [comment]
@maxbeatty Is this a regression in v0.16? Do you know if it worked in previous versions?

**Comment by maxbeatty** [comment]
We just launched with v0.16. I cloned the environment in AWS and uploaded v0.15.1. That version connects to the database fine and loads the tables, but I can't see any columns. Do you have any advice to get the columns to display? (AWS Elastic Beanstalk environment: 64bit Amazon Linux 2015.09 v2.0.8 running Docker 1.9.1 with postgres 9.4.5)

**Comment by maxbeatty** [comment]
In v0.16, questions written as SQL display time when selecting "timestamp with time zone" fields

```
select t.date from table as t;
```

| date |
| ---- |
| Sunday, April 3, 2016 12:10 AM |

**Comment by agilliland** [comment]
Hey @maxbeatty , for your latest comment I'm not sure I understand what seems off to you.  What were you expecting to see?

**Comment by agilliland** [comment]
Took a quick look at reproducing this and I'm not seeing any problems.  We use postgres for a lot of our development work and I've never seen this problem occur, but here's a test case ...

```
create table ts(ts timestamp with time zone);
insert into ts values(now());
insert into ts values(now());
insert into ts values(now());
```

then in Metabase

```
select * from ts
```

which yields this output for me ...

<img width="276" alt="screen shot 2016-04-04 at 10 48 56 am" src="https://cloud.githubusercontent.com/assets/1987787/14257496/e1e57be2-fa52-11e5-858b-d0413e86315d.png">

<img width="245" alt="screen shot 2016-04-04 at 10 48 49 am" src="https://cloud.githubusercontent.com/assets/1987787/14257501/e69e15f4-fa52-11e5-9032-9784026ba157.png">

**Comment by maxbeatty** [comment]
@agilliland the raw SQL query displays the time as expected but when building a query through the UI, those same columns do not display the time, only the day:

<img width="124" alt="screen shot 2016-04-04 at 11 11 28 am" src="https://cloud.githubusercontent.com/assets/93179/14258164/0b185e96-fa56-11e5-8464-613c08ce0ba9.png">

How do I get timestamp with time zone columns to display the full date with time when building queries through the UI?

**Comment by camsaul** [comment]
Fixed by #2306
